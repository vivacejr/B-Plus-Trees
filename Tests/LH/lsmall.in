6
7
4
5
2
100
5
6
3
2
1
10
56
43
3
12
100