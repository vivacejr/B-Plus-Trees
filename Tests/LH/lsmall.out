6
7
4
5
2
100
3
1
10
56
43
12
